staging/src/k8s.io/apiextensions-apiserver/artifacts/example/rc.yaml

```diff
@@ -23,4 +23,4 @@ spec:
         - "--etcd-servers=http://localhost:2379"
         - "--audit-log-path=-"
       - name: etcd
-        image: quay.io/coreos/etcd:v3.1.10
+        image: quay.io/coreos/etcd:v3.2.13
```

Comment:
Should this be 3.2.14?

Reply:
Thanks for catching! This is what I get for starting with an upgrade to 3.2.13 and then switching to a upgrade to 3.2.14 after creating the initial PR.